service/service.go

```diff
@@ -0,0 +1,53 @@
+package service
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"net/url"
+
+	goaclient "github.com/goadesign/goa/client"
+
+	"github.com/fabric8-services/fabric8-common/errors"
+	"github.com/fabric8-services/fabric8-common/goasupport"
+
+	clusterclient "github.com/fabric8-services/fabric8-cluster-client/cluster"
+)
+
+type Service interface {
+	UserClusters(ctx context.Context) (*clusterclient.ClusterList, error)
+}
+
+func NewClusterService(clusterURL string) (Service, error) {
+	url, err := url.Parse(clusterURL)
+	if err != nil {
+		return nil, err
+	}
+	return &service{url}, nil
+}
+
+type service struct {
+	clusterURL *url.URL
+}
+
+func (s *service) createClient(ctx context.Context) *clusterclient.Client {
+	c := clusterclient.New(goaclient.HTTPClientDoer(http.DefaultClient))
+	c.Host = s.clusterURL.Host
+	c.Scheme = s.clusterURL.Scheme
+	c.SetJWTSigner(goasupport.NewForwardSigner(ctx))
+	return c
+}
+
+// ClustersUser takes context with user token and returned linked clusters for given user token
```

Comment:
you need to update the comment here, since you renamed the method. Also, it could be  something like `takes a context with a user token and returns the clusters linked to the corresponding user`

Reply:
Ok